toolkit/tools/internal/retry/retry_test.go

```diff
@@ -0,0 +1,83 @@
+package retry
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// Delay vs. retry count for mult = 1.0, delay = .1 seconds
+var expectedValuesMillis1 = []int{0, 100, 200, 300, 400, 500}
```

Comment:
Let's keep the expected results inside the tests, which use them - that way it's easier to find all of the test's parameters because they are in one place. In this case we're using `expectedValuesMillis1` only once, so we don't even benefit from keeping it a global.